src/components/common/PostalCode/postal-code.scss

```diff
@@ -0,0 +1,50 @@
+@import "src/styles/scaffold";
+
+.postal-code-input {
+  // Lighthouse
+  margin-top: 1rem;
+
+  [data-store-label] {
+    font-family: Arial; /* stylelint-disable-line */ // Prevent layout shift
+  }
+
+  [data-store-input] {
+    border: var(--fs-border-width-default) solid var(--fs-border-color-default);
+    border-radius: var(--fs-border-radius-default);
+
+    // Lighthouse
+    height: var(--fs-control-tap-size); /* stylelint-disable-line */
+    margin-left: 1rem;
+    font-family: Arial; /* stylelint-disable-line */ // prevent layout shift
+  }
+}
+
+[data-postal-code-bar] {
+  width: 100%;
+
+  .button[data-store-button] {
+    width: 100%;
+    padding: 0 var(--fs-spacing-3) 0 var(--fs-spacing-4);
+    color: var(--fs-color-text-display);
+    background-color: var(--fs-color-neutral-0);
+    border-radius: 0;
+
+    @include media(">=notebook") {
+      min-height: var(--fs-spacing-5);
+      padding: var(--fs-spacing-0) var(--fs-spacing-3) var(--fs-spacing-0) 0;
+      border-right: 1px solid var(--fs-border-color-light);
+    }
+  }
+
+  span {
+    &:first-of-type {
+      margin-right: auto;
+      font-weight: var(--fs-text-weight-regular);
```

Comment:
```suggestion
      font-weight: var(--fs-text-weight-regular);
      
      @include media(">=notebook") {
         font-weight: var(--fs-text-weight-bold);
       }
```

Reply:
It looks strange to me... What do you think, @filipewl @hellofanny?
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/22377378/161542600-6780bcbe-dd58-45e7-8000-a916194c06bf.png">

Reply:
Done.